src/main/java/nl/knaw/dans/pf/language/ddm/relationhandlers/DcRelationHandler.java

```diff
@@ -23,9 +23,6 @@
     @Override
     public void finishElement(final String uri, final String localName) throws SAXException {
         final BasicString relation = createBasicString(uri, localName);
-        final String scheme = getAttribute("", "scheme");
```

Comment:
@lindareijnhoudt now that I look at this again, I'm wondering whether this removal is actually OK. I use this handler for `dc:relation` and `dcterms:relation` now, not anymore for `ddm:relation`. Is it true that the former two can't have this `scheme` attribute?